fdbserver/OldTLogServer_6_2.actor.cpp

```diff
@@ -323,6 +323,7 @@ struct TLogData : NonCopyable {
 	int64_t instanceID;
 	int64_t bytesInput;
 	int64_t bytesDurable;
+	int64_t targetVolatileBytes;
```

Comment:
Maybe add a comment explaining the meaning. It's not obvious to me what this is.

Reply:
```suggestion
	int64_t targetVolatileBytes;  // The number of bytes of mutations this TLog should hold in memory before spilling.
```